### PR TITLE
Add info about fp8 support

### PR DESCRIPTION
**Pull request by jkaniecki**
Update README file with fp8 instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ limitations under the License.
 
 - [Running TGI on Gaudi](#running-tgi-on-gaudi)
 - [Adjusting TGI parameters](#adjusting-tgi-parameters)
+- [Running TGI with fp8 precision](#running-tgi-with-fp8-precision)
```
Comment (kdamaszk) [inline README.md]: Please change to "FP8" everywhere, like in https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation#running-with-fp8

File: README.md [hunk 2]
```diff
@@ -91,23 +92,33 @@ Except those already mentioned, there are other parameters that need to be prope
 
 For more information and documentation about Text Generation Inference, checkout [the README](https://github.com/huggingface/text-generation-inference#text-generation-inference) of the original repo.
 
+## Running TGI with fp8 precision
+
+TGI supports fp8 precision runs within the limits provided by [Habana Quantization Toolkit](https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html). Models with fp8 can be ran by properly setting QUANT_CONFIG environment variable. Deatiled instruction on how to use that variable can be found in [Optimum Habana fp8 guide](https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation#running-with-fp8). Summarising that instruction in TGI cases:
+
+1. Measure quantization statistics of requested model by using [Optimum Habana measurement script](https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation#running-with-fp8:~:text=use_deepspeed%20%2D%2Dworld_size%208-,run_lm_eval.py,-%5C%0A%2Do%20acc_70b_bs1_measure.txt)
+2. Run requested model in TGI with proper QUANT_CONFIG setting - f. e. `QUANT_CONFIG=./quantization_config/maxabs_quant.json`
```
Comment (kdamaszk) [inline README.md]: f. e. ?

File: README.md [hunk 3]
```diff
@@ -91,23 +92,33 @@ Except those already mentioned, there are other parameters that need to be prope
 
 For more information and documentation about Text Generation Inference, checkout [the README](https://github.com/huggingface/text-generation-inference#text-generation-inference) of the original repo.
 
+## Running TGI with fp8 precision
+
+TGI supports fp8 precision runs within the limits provided by [Habana Quantization Toolkit](https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html). Models with fp8 can be ran by properly setting QUANT_CONFIG environment variable. Deatiled instruction on how to use that variable can be found in [Optimum Habana fp8 guide](https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation#running-with-fp8). Summarising that instruction in TGI cases:
```
Comment (kdamaszk) [inline README.md]: "Detailed" typo

File: README.md (hunk 1 above)
Comment (jkaniecki) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (jkaniecki) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (jkaniecki) [reply README.md]: Fixed

**Review by kdamaszk — APPROVED** [review]

**Pull request merged**